### PR TITLE
Cleanup Trash & Remove Dune V1

### PR DESCRIPTION
**Pull request by bh2smith**
We no longer use Dune V1 and can remove everything associated with it. Unfortunately this also means removing much of our postgres testing framework -- but its outdated anyway.

### CODE REVIEW

File: .github/workflows/postgres.yaml [hunk 1]
```diff
@@ -29,9 +29,7 @@ jobs:
       - name: Install Requirements
         run: pip install -r requirements.txt
       - name: Tests
-        run: |
-          python -m pytest tests/db
```
Comment (gentrexha) [inline .github/workflows/postgres.yaml:33]: Nit: You still have `tests/db` in your `.dockerignore` file.

Comment (gentrexha) [inline .github/workflows/postgres.yaml:33]: Nit: You still have `tests/db` in your `.dockerignore` file.

Not related to this file, just didn't know where else to put this comment.

File: hooks/pre-push
Comment (gentrexha) [inline hooks/pre-push:1]: Thanks for removing those!

**Review by gentrexha — CHANGES_REQUESTED** [review]
There's still a lot of Dune V1 code in the integration tests.

**Comment by bh2smith** [comment]
>  Dune V1 code in the integration tests.

I will go and check this out thanks!

File: hooks/pre-push
Comment (bh2smith) [reply hooks/pre-push:1]: Yea -- they were a cool idea but never got used for anything. I think our `make check` is a pretty good alternative.

File: .github/workflows/postgres.yaml (hunk 1 above)
Comment (bh2smith) [reply .github/workflows/postgres.yaml:33]: Fixed.

**Review by gentrexha — COMMENTED** [review]
You're still using a `dune_v1` query inside of `src/models/slippage.py`. The fact that we don't have a test that catches that is worrying.

**Comment by bh2smith** [comment]
> The fact that we don't have a test that catches that is worrying.

This appears to be a python problem that does not notify us of unused functions. Very good eye on this. I have found two things that are never used anywhere.

Note that the next step after this is to remove all "Pre-CIP20" code which will also eliminate a huge chunk of no longer used logic.

**Review by gentrexha — APPROVED** [review]

**Pull request merged**